vms/platformvm/txs/executor/staker_tx_verification_test.go

```diff
@@ -158,8 +154,9 @@ func TestVerifyAddPermissionlessValidatorTx(t *testing.T) {
 				return &Backend{
 					Ctx: ctx,
 					Config: &config.Config{
```

Comment:
Why aren't we using `defaultTestConfig` here?

Reply:
Fixed. Thanks for spotting this.